app/assets/javascripts/admin/loan_questionnaires_view.coffee

```diff
@@ -32,6 +32,16 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
     'click .edit-rt-response': 'openRichTextModal'
     'click a.deep-link': 'scrollElementToView'
     'change input': 'formUpdated'
+    'click .help-block .expansion-control': 'toggleExpansion'
+
```

Comment:
expanding/collapsing instructions is different than other expand/collapse in madeline. The difference is that rather than hiding/showing and element, we are changing the styles on the element. We change the styles on the element because we're previewing the top line, and putting 'the rest' into a separate element is extra, messy work.